### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by AdrianCSM**
# Features : 
- Add books [WIP]
- Remove books [WIP]

### CODE REVIEW

**Review by ahmetbozaci — COMMENTED** [review]
# STATUS: INVALID  ❌ 


### Hello @AdrianCSM  ✋🏼 
#### You have submitted a project, that is using wrong flow. You should use git-flow not github flow. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=Make%20sure%20that%20you%20used%20correct%20flow%20Gitflow.).

#### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏


### Cheers and Happy coding! 💻 👏

coding

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

_Invalid Code Review Request **does not count** into the code reviews limit._

___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
-An online bookstore built with react.
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```
Comment (GzimAsani) [inline README.md]: - [x] Please explain to the users how can they clone and run your project in details

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @AdrianCSM 
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Your code organization is good.
 
 ## Required Changes 
 - Please explain better(in more detail) the steps that a user should take in order to clone and run your project




 _Check the comments under the review._
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AdrianCSM) [reply README.md]: Alright, forgot to change the README.md file. Thanks for pointing out.

Comment (GzimAsani) [reply README.md]: - [x] Please be descriptive about how users can clone and run your project. Make it step by step.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,65 @@
...
-An online bookstore built with react.
+# Bookstore
+
+This is a single page app (SPA) that allows you to add, remove and filter books.
+
+---
+
+## Live Demo
+
+[Link to the live demo](https://adriancsm.github.io/Bookstore-React/) - WIP
```
Comment (GzimAsani) [inline README.md]: - [x] This link is broken, kindly update or remove it

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @AdrianCSM 
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 

 ## Required Changes 
 - Please explain better(in more detail) the steps that a user should take in order to clone and run your project




 _Check the comments under the review._
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — APPROVED** [review]
Hi @AdrianCSM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**